google/ggrc-core
# GGRC-4463 Remove 'Dropdown' option of TG and CTG tasks

# Issue description

Remove TaskGroupTask and CycleTask type 'Dropdown'

# Steps to test the changes
1. Log into ggrc app
2. Start a Workflow
3. Create two or more tasks under the Workflow's Task group. While creating tasks, 'Dropdown' option for task type should not be available.
4. After saving the tasks, try editing them and the edit page should not contain 'Dropdown' option for task type.
5. Activate the workflow
6. Try to edit 'Cycle task group tasks', 'Dropdown' option should not be available there too.
7. Also check in 'Cycle task' info pane, 'Dropdown' option should not be visible.

# Solution description

* Removed 'Dropdown' option in files 'src/ggrc-client/js/templates/task_group_tasks/modal_content.stache' and 'src/ggrc-client/js/templates/cycle_task_group_object_tasks/modal_content.stache' 
* Also modified the definition of 'TaskGrouptask' model in 'src/ggrc_workflows/models/task_group_task.py' file.

# Sanity checklist




# PR Review checklist
